portfolio/src/main/webapp/script.js

```diff
@@ -150,6 +151,47 @@ function getComments()
     });
 }
 
+/** Check the login status to display the comments or force sign in */
+function checkLogin()
```

Comment:
Function name does not fully reflects what is is doing. 
Maybe setLogInOutButton ?